docs/ARCHIVING.md

```diff
@@ -136,6 +136,9 @@ NOTE: At the time of writing, this is a completely manual process. This is becau
 
 ## Build 0.9 sub-site
```

Comment:
This revised process was tested and validated. The result is live on archive.docs.dagger.io